docs/probes.md

```diff
@@ -26,105 +26,114 @@ Make sure to provide enough delay and failureThreshold for the VM and the agent
+> manifest shown below. Otherwise, the `chcon`  command will fail.
 
 1. Create VM manifest
 
 ```yaml
-# /tmp/probe-test.vm.yaml
-apiVersion: kubevirt.io/v1alpha3
+# /tmp/probe-test.yaml
+apiVersion: kubevirt.io/v1
 kind: VirtualMachine
 metadata:
-  name: probe-test
-  namespace: default
   labels:
-    app: probe-test
-    vm.kubevirt.io/name: probe-test
-    kubevirt.io/domain: probe-test
+    kubevirt.io/vm: readiness-probe-vm
+  name: readiness-probe
 spec:
   running: false
   template:
     metadata:
       labels:
-        vm.kubevirt.io/name: probe-test
-        kubevirt.io/domain: probe-test
+        kubevirt.io/domain: readiness-probe
+        kubevirt.io/vm: readiness-probe
     spec:
-      readinessProbe:
-        exec:
-          command:
-          - cat
-          - /tmp/ready.txt
-        failureThreshold: 10
-        initialDelaySeconds: 120
-        periodSeconds: 10
-        # Note that timeoutSeconds value does not have any impact before K8s v1.20.
-        timeoutSeconds: 5
       domain:
         cpu:
           cores: 1
           sockets: 1
           threads: 1
         devices:
           disks:
-            - disk:
+            - name: containerdisk
+              disk:
                 bus: virtio
-              name: cloudinitdisk
-            - bootOrder: 1
+            - name: cloudinitdisk
               disk:
                 bus: virtio
-              name: rootdisk
           interfaces:
-            - masquerade: {}
-              model: virtio
-              name: nic-0
-          networkInterfaceMultiqueue: true
+            - name: default
+              masquerade: {}
           rng: {}
         machine:
           type: pc-q35-rhel8.2.0
         resources:
           requests:
             memory: 1Gi
-      hostname: probe-test
       networks:
-        - name: nic-0
+        - name: default
           pod: {}
+      readinessProbe:
+        exec:
+          command: ["cat", "/tmp/healthy.txt"]
+        failureThreshold: 10
+        initialDelaySeconds: 120
+        periodSeconds: 10
+        # Note that timeoutSeconds value does not have any impact before K8s v1.20.
+        timeoutSeconds: 5
       terminationGracePeriodSeconds: 180
       volumes:
+        - containerDisk:
+            image: quay.io/containerdisks/fedora
+          name: containerdisk
         - cloudInitNoCloud:
-            userData: |
+            userData: |-
               #cloud-config
-              user: fedora
-              password: fedora
               chpasswd:
                 expire: false
+              password: password
+              user: fedora
               packages:
                 qemu-guest-agent
               runcmd:
-                - [ "systemctl", "enable", "--now", "qemu-guest-agent" ]
-                - [ "touch" "/tmp/ready.txt" ]
+                - ["touch", "/tmp/healthy.txt"]
+                - ["sudo", "chcon", "--type", "virt_qemu_ga_exec_t", "/tmp/healthy.txt"]
```

Comment:
I get listing this as a caveat but if it's already installed in the referenced Fedora image then why add the commands here?

Reply:
I'm not sure if I fully understand your question. But without the `chcon` command, the probe will fail because SELinux will deny attempts to open the `/tmp/healthy.txt` because of the context. Sorry, maybe this is not what you are asking.

Reply:
I think I've misunderstood your following comment at the top of the file:

> Also, cloud-init assigns the proper SELinux context, i.e. virt_qemu_ga_exec_t, to the `/tmp/healthy.txt` file. 

I assumed you were saying `cloud-init` was doing this for us automatically somehow.

Reply:
Maybe I can extend a bit this explanation to make it cleaner.